architecture/core_objects/builds.adoc

```diff
@@ -172,41 +174,19 @@ can also add authentication entries to this file by running the `docker login`
 command. The file will be created if it does not exist.
 
 Kubernetes provides the
-https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md[Secret]
-resource, which is used to store your configuration and passwords. You must
-first create the `*Secret*` before builds can use your *_.dockercfg_* file for
-pushing the output image:
```

Comment:
@bparees I nuked the second part because that is not longer true... You have builder service account that search all .dockercfg secrets you have and try to find the valid one...